src/index.ts

```diff
@@ -117,9 +120,13 @@ import { isNotEmptyString } from "utils";
     consumer.subscribe({
       topic: RDPC_PROGRAM_UPDATE_TOPIC,
     }),
+    consumer.subscribe({
+      topic: KAFKA_PUBLIC_RELEASE_TOPIC,
+    }),
   ]);
   logger.info(`subscribed to source events ${CLINICAL_PROGRAM_UPDATE_TOPIC}`);
   logger.info(`subscribed to source events ${RDPC_PROGRAM_UPDATE_TOPIC}`);
+  logger.info(`subscribed to source events ${KAFKA_PUBLIC_RELEASE_TOPIC}`);
```

Comment:
These logs occur only once ALL are subscribed to at, so let's simplify log to a list:
```
logger.info(`subscribed to source events ${[CLINICAL_PROGRAM_UPDATE_TOPIC, RDPC_PROGRAM_UPDATE_TOPIC, KAFKA_PUBLIC_RELEASE_TOPIC].join(', ')}`)
```

But more importantly... do you know what happens if one of the consumer subscribe fails? we have no catch here, do we just fail to start up or do we log the issue?